platform-common/src/test/java/com/softicar/platform/common/core/exception/ExceptionsCollectorTest.java

```diff
@@ -1,6 +1,8 @@
 package com.softicar.platform.common.core.exception;
 
+import com.softicar.platform.common.core.logging.Log;
 import com.softicar.platform.common.testing.AbstractTest;
+import java.util.List;
 import org.junit.Test;
 
 public class ExceptionsCollectorTest extends AbstractTest {
```

Comment:
This test focuses on `Exception` while the `ExceptionsCollector` API uses `Throwable`. That's odd.

Reply:
so what to do? I don't want to talk about `Throwable` when most people only know about `Exception`

Reply:
Well I guess we can't do much as long as we uphold the assumption about ppl being ignorant towards the existence of `Throwable`. Feel free to close this.